packages/react-components/lib/lifts/tests/lift-accordion.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { LiftAccordion } from '..';
+import { makeLift } from './test-utils';
+
+test(`smoke test`, () => {
```

Comment:
Maybe we can add a snapshot too.

Reply:
I did do snapshot testing at first, but material-ui is not designed with snapshot testing in mind and they do not recommend it. You would need to go through some hacks to make the components stable under snapshots, I think it's probably not worth the trouble.